### ISSUE TITLE
Modular Blueprints

### ISSUE THREAD

**Issue by loranger** [getgrav/grav-theme-quark#144]
I don't know if Quark theme repository is the right place for this issue, but as far as I'm extending Quark theme, here I come :

I was wondering why there is no `blueprints/modular.yaml`, and therefore why a toggle for the `onpage_menu` option is not provided whereas this should be really convenient.

I did create such a blueprint for my themes, but I don't know if I should provide the code as a PR and if I should provide it here or at a _grav level_

Thank, by the way for this awesome tool an theme !

**Comment by DeFUCC** [comment]
Yes, I can't understand how the modular page is defined in the theme without `blueprints/modular.yaml`. I also see jQuery treemenu plugin, but can't figure out how can it be used...

**Comment by pmoreno-rodriguez** [comment]
Hi. For the next version I will try to add this option.

**Comment by pmoreno-rodriguez** [comment]
Ready in version 2.1.0
